includes/Jobs/CleanupSkyvergeFrameworkJobOptions.php

```diff
@@ -51,7 +50,6 @@ public function clean_up_old_completed_options() {
 			"DELETE
 			FROM {$wpdb->options}
 			WHERE option_name LIKE 'wc_facebook_background_product_sync_job_%'
-			AND option_value LIKE '%\"status\":\"completed\"%'
```

Comment:
I have discussed this with @budzanowski. 

> The options table should not be used to store logs this is a correct comment from the Issue.

This seems to be an artifact of the SkyVerge framework. We have never used those logs for anything.  I couldn't find any usage of this option's value in the code. 

Maybe removing everything except for items currently processing would be less drastic :-D. 

@budzanowski, any thoughts?

Reply:
@rawdreeg, @mikkamp is right. We only want to clear completed and failed. Queued and processing are not permanent, we don't have a problem with them - they are not persistent in the options table. They are crucial for ongoing plugin operations though.
The cycle is:

`queued -> processing -> completed || failed`

You could probably increase also the limit because we have seen that 250 is both not a problem for the DB query and also sometimes it is not enough. What do you think about 500? Should it still be not a problem?

Reply:
Thanks, @mikkamp, @budzanowski 

You are correct; the status of the stale job will either be "completed" or "failed." I have modified the query to catch failed jobs accordingly.

I have also raised the limit to 500. Unfortunately, I cannot run a benchmark on this, but I think that should be fine still. 

R